src/pvrclient-nextpvr.cpp

```diff
@@ -461,7 +461,7 @@ PVR_ERROR cPVRClientNextPVR::GetEpg(ADDON_HANDLE handle, const PVR_CHANNEL &chan
 
         broadcast.iUniqueBroadcastId  = atoi(pListingNode->FirstChildElement("id")->FirstChild()->Value());
         broadcast.strTitle            = title;
-        broadcast.iChannelNumber      = channel.iChannelNumber;
+		broadcast.iUniqueChannelId    = channel.iUniqueId;
```

Comment:
Whitespace

Reply:
It looks like you've used tabs as whitespace while the rest of the code uses two spaces. Do you think you could change that?